<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mm_against">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>联机对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cm_against">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>人机对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_room">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>创建房间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="join_room">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>加入房间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>输入加入的房间号（仅限数字）</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
